packages/components/src/local/molecules/chat-message-bubble/types/props.d.ts

```diff
@@ -0,0 +1,47 @@
+export default interface Props {
+  /**
+   * Header item title
+   */
+  title?: string
+  /**
+   * Message timestamp
+   */
+  timestamp?: string
+  /**
+   * Message sender role
+   */
+  role?: string
+  /**
+   * Message read state
+   */
+  hasBeenRead?: boolean
+  /**
```

Comment:
if we're not using authorForceID and playerForceId properties they should be deleted.